content/en/apps/examples/covid-rdt-reference-app.md

```diff
@@ -0,0 +1,521 @@
+
+<!-- 
+  use raw HTML for now as hugo youtube short codes show 16:9 ration 
+  box instead of mobile portrait ~9:16
+-->
+<iframe id="ytplayer" type="text/html" width="290" height="610" src="https://www.youtube.com/embed/FlhZ-KQIfdY" frameborder="0"></iframe>
+
+#### Capture
+
+This second video shows the right side of the workflow above to capture RDT results for a patient.  The CHW is shown viewing the "Capture Covid-19 RDT" task after the 15 minute timer from the RD-Toolkit app has completed. After selecting the task, the CHW is brought to the CHT form which has the session information for the RD-Toolkit already loaded.  The test results are recorded and then returned to the CHT from the RD-Toolkit. The last part of the video shows the completed capture report in the CHT, including the result and image of the RDT:
+
+<!-- 
+  use raw HTML for now as hugo youtube short codes show 16:9 ration 
+  box instead of mobile portrait ~9:16
+-->
+<iframe id="ytplayer" type="text/html" width="290" height="610" src="https://www.youtube.com/embed/LC9gR3N8o80" frameborder="0"></iframe>
+
+## Reports and Dashboards
+
+Like all applications written for the CHT, there are built in mechanism to retrieve raw and aggregate data to generate reports and dashboards. Here are some ways that the data can be accessed:
+
+ * **[In app targets]({{< ref "apps/features/targets" >}}):** Gives the CHW or their supervisor an aggregate view of any of the form fields. Since targets rely on the data on the device, if targets include data from other users then permissions must be set on the relevant forms so that the data can be replicated and synchronized accordingly.
+ * **API Calls:** Given that all form submissions are captured in JSON and that we know the data model well, you can easily do API calls to a CHT server instance and use some custom code (node, python etc) to gather and show stats on a daily basis. You can export to either JSON or CSV. See API docs [for reports]({{< ref "apps/reference/api#get-apiv2exportreports" >}}) as well as [monitoring metadata]({{< ref "apps/reference/api#get-apiv2monitoring" >}}). 
+ * **[Postgres queries](https://github.com/medic/cht-couch2pg):** CHT ships with a utility to export all the data that the API has to a relational database, Postgres. You have all the raw data the API has, but can now use the power of joins and groupings to come up with totally customizable stats by day, week, month etc. Data can be synched daily or hourly from the CHT. 
+ * **Dashboards:** We have used both [Klipfolio](https://www.klipfolio.com/) and, more recently, [Superset](https://superset.incubator.apache.org/) to create more complex yet still user-friendly dashboards. This is particular useful for those who need to view the data but wouldn't otherwise be logging in to CHT apps. These dashboards generally access the relational data in the Postgres database as the back end.
```

Comment:
~We~

```suggestion
 * **Dashboards:** Medic has used both [Klipfolio](https://www.klipfolio.com/) and, more recently, [Superset](https://superset.incubator.apache.org/) to create more complex yet still user-friendly dashboards. This is particular useful for those who need to view the data but wouldn't otherwise be logging in to CHT apps. These dashboards generally access the relational data in the Postgres database as the back end.
```